schema/schema.json

```diff
@@ -0,0 +1,122 @@
+{
+    "description": "Schema for OpenContainer bundle configuration file",
```

Comment:
I recommend using config.json, etc. in the schema directory, so we can also add schemas for validating state JSON, etc.